### PR TITLE
Used oVirt.engine-setup role for engine setup

### PR DESCRIPTION
**Pull request by gobindadas**

### CODE REVIEW

File: playbooks/hc-ansible-deployment/he_gluster_vars.json [hunk 1]
```diff
@@ -11,5 +11,6 @@
   "he_storage_domain_path": "/engine",
   "he_storage_domain_addr": "host1",
   "he_mount_options": "backup-volfile-servers=host2:host3",
-  "he_bridge_if": "bridge name"
+  "he_bridge_if": "bridge name",
+  "ovirt_repo_release_rpm": "http://resources.ovirt.org/pub/yum-repo/ovirt-release42-snapshot.rpm"
```
Comment (sabose) [inline playbooks/hc-ansible-deployment/he_gluster_vars.json]: why is this needed? What happens if left empty?

File: playbooks/hc-ansible-deployment/tasks/he_deployment.yml [hunk 2]
```diff
@@ -1,23 +1,48 @@
...
-- name: Pre Installation Steps
+- name: Install packages and bootstrap local engine VM
   hosts: localhost
   connection: local
   vars:
     he_install_packages: true
     he_pre_checks: true
     he_initial_clean: true
     he_bootstrap_local_vm: true
+    ovirt_repositories_ovirt_release_rpm: "{{ ovirt_repo_release_rpm }}"
```
Comment (sabose) [inline playbooks/hc-ansible-deployment/tasks/he_deployment.yml]: can package installation be done only when the release rpm is provided. For instance, on node - we do not need pkg installation

File: playbooks/hc-ansible-deployment/he_gluster_vars.json (hunk 1 above)
Comment (sabose) [reply playbooks/hc-ansible-deployment/he_gluster_vars.json]: Not the he_bridge_if but ovirt_repo_release_rpm - is this a mandatory variable?

File: playbooks/hc-ansible-deployment/tasks/he_deployment.yml (hunk 2 above)
Comment (gobindadas) [reply playbooks/hc-ansible-deployment/tasks/he_deployment.yml]: For node which doesn't require pkg to be installed then ovirt_repositories_ovirt_release_rpm is not required.This is optional.

File: playbooks/hc-ansible-deployment/he_gluster_vars.json (hunk 1 above)
Comment (gobindadas) [reply playbooks/hc-ansible-deployment/he_gluster_vars.json]: No it's not mandatory

File: playbooks/hc-ansible-deployment/README [hunk 3]
```diff
@@ -14,3 +14,19 @@ For gluster deployment clean up:
...
+"ovirt_repositories_ovirt_release_rpm": "http://resources.ovirt.org/pub/yum-repo/ovirt-master-release.rpm"
+"ovirt_repositories_use_subscription_manager": false,
+"ovirt_repositories_ovirt_version": 4.2,
+"ovirt_repositories_target_host": "host1",
+"ovirt_repositories_rh_username": "Username to use for subscription manager",
+"ovirt_repositories_rh_password": "Password to use for subscription manager",
+"ovirt_repositories_pool_ids": ["List of pools ids to subscribe to"],
+"ovirt_repositories_pools": ["List of subscription pool names"],
+"ovirt_repositories_repos_backup_path": "Directory to backup the original repositories configuration",
+"ovirt_repositories_force_register": false,
```
Comment (sabose) [inline playbooks/hc-ansible-deployment/README]: Can we comment these by default?

File: playbooks/hc-ansible-deployment/tasks/he_deployment.yml (hunk 2 above)
Comment (sabose) [reply playbooks/hc-ansible-deployment/tasks/he_deployment.yml]: So if that variable is not defined, does this playbook need to be edited?

**Comment by gobindadas** [comment]
No we don't have to change playbook, role will be executed only when variable is defined.

**Comment by sabose** [comment]
Ok! so ack from me
